hubgrep/lib/sphinx.py

```diff
@@ -72,11 +72,9 @@ def _make_sql_time_filter(
             time_filter_vars.append(int(created_before.timestamp()))
         if updated_after:
             # we dont make a difference if its just the project meta or the repo itself that was updated.
-            # since its two times the same var used, we append it twice.
-            time_filters.append("(updated_at >= %s or pushed_at >= %s)")
+            # (also, sphinx doesnt supprt "or", so we have a special attribute...)
+            time_filters.append("pushed_or_updated_at >= %s")
```

Comment:
that was a weird one. sphinx doesnt support "or" for attributes, so i made a new one. 
`pushed_or_updated_at` contains the biggest of both numbers...

Reply:
todo: split up filter to updated and pushed_at